Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -76,6 +76,10 @@ lemma traceForm_apply_apply (x y : L) :
 lemma traceForm_comm (x y : L) : traceForm R L M x y = traceForm R L M y x :=
   LinearMap.trace_mul_comm R (φ x) (φ y)
 
+lemma traceForm_isSymm : LinearMap.IsSymm (traceForm R L M) := by
+  intro x y
+  rw [RingHom.id_apply, LieModule.traceForm_comm]
```

Comment:
This is actually true by definition:
```suggestion
lemma traceForm_isSymm : LinearMap.IsSymm (traceForm R L M) := LieModule.traceForm_comm R L M
```
which makes me think we should drop `traceForm_comm` and just use this new `traceForm_isSymm` everywhere instead. (Not necessary as part of these changes.)

Reply:
Ok. I will prepare a dedicated commit once we merge this one.